lib/rules/lint-no-unnecessary-concat.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Rule = require('./base');
+
+module.exports = class NoInputBlock extends Rule {
```

Comment:
The class name here is left over from copy pasta...

Reply:
done